src/unicodeSymbols.js

```diff
@@ -1,323 +1,30 @@
-// @flow
```

Comment:
Could we keep the `// @flow` and explicitly type `const result = {};`?

Reply:
@ylemkimon Yes, but as this file is not transpiled, Flow comment types syntax should be used.